### PR TITLE
Add ospf junos resource module

### PR DESCRIPTION
**Pull request by danielmellado**
Signed-off-by: Daniel Mellado <dmellado@redhat.com>

### CODE REVIEW

File: plugins/modules/junos_ospf.py [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "network",
+}
+
+DOCUMENTATION = """
+---
+module: junos_ospf
+version_added: "2.10"
+short_description: Manages OSPFv2 configuration on devices running Juniper JUNOS.
```
Comment (trishnaguha) [inline plugins/modules/junos_ospf.py:43]: Please append the keyword "OSPFV2 resource module" at the end of the short_description.

**Comment by danielmellado** [comment]
recheck

**Comment by danielmellado** [comment]
recheck

**Comment by danielmellado** [comment]
recheck

**Comment by danielmellado** [comment]
recheck

**Comment by danielmellado** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Pull request merged**